app/_components/pdf-viewer.tsx

```diff
@@ -72,12 +75,17 @@ export default function PdfViewer({
 
   function onPageLoadSuccess(pdfProxy: PDFPageProxy) {
     const scale = (pdfProxy.userUnit || 1) * PDF_TO_CSS_UNITS;
+    const width = pdfProxy.view[2] * scale;
+    const height = pdfProxy.view[3] * scale;
     setPageSize({
       action: "setPage",
       pageNumber: pdfProxy.pageNumber,
       width: pdfProxy.view[2] * scale,
       height: pdfProxy.view[3] * scale,
```

Comment:
Super nit, but we can reuse the `width` and `height` values from above :)
```suggestion
      width,
      height,
```

Reply:
> Super nit, but we can reuse the `width` and `height` values from above :)

You're absolutely right! I have made the change. I apologize for the late reply; I was on holiday and unable to address this sooner. I also rebased the branch, resolving some conflicts. Please review it to ensure my changes are still correct :smiley: